From patchwork Thu Apr 26 06:43:06 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [U-Boot] rockchip: make_fit_atf: fix warning unit_address_vs_reg
X-Patchwork-Submitter: Kever Yang <kever.yang@rock-chips.com>
X-Patchwork-Id: 904884
Message-Id: <1524724986-28497-1-git-send-email-kever.yang@rock-chips.com>
To: u-boot@lists.denx.de
Date: Thu, 26 Apr 2018 14:43:06 +0800
From: Kever Yang <kever.yang@rock-chips.com>
List-Id: U-Boot discussion <u-boot.lists.denx.de>

Patch fix warning:
/builddir/BUILD/u-boot-2018.05-rc2/"arch/arm/mach-rockchip/make_fit_atf.py" \
arch/arm/dts/rk3399-firefly.dtb > u-boot.its
  ./tools/mkimage  -f u-boot.its -E u-boot.itb >/dev/null  && cat
/dev/null
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node /images/uboot@1
has a unit name, but no reg property
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node /images/atf@1 has
a unit name, but no reg property
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node /images/atf@2 has
a unit name, but no reg property
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node /images/atf@3 has
a unit name, but no reg property
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node /images/fdt@1 has
a unit name, but no reg property
u-boot.itb.tmp: Warning (unit_address_vs_reg): Node
/configurations/config@1 has a unit name, but no reg property
make[1]: Leaving directory
'/builddir/BUILD/u-boot-2018.05-rc2/builds/firefly-rk3399'

Reported-by: Peter Robinson <pbrobinson@gmail.com>
Signed-off-by: Kever Yang <kever.yang@rock-chips.com>
---

 arch/arm/mach-rockchip/make_fit_atf.py | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm/mach-rockchip/make_fit_atf.py b/arch/arm/mach-rockchip/make_fit_atf.py
index 7c6dd57..513e088 100755
--- a/arch/arm/mach-rockchip/make_fit_atf.py
+++ b/arch/arm/mach-rockchip/make_fit_atf.py
@@ -37,7 +37,7 @@ DT_HEADER="""/*
 	#address-cells = <1>;
 
 	images {
-		uboot@1 {
+		uboot {
 			description = "U-Boot (64-bit)";
 			data = /incbin/("u-boot-nodtb.bin");
 			type = "standalone";
@@ -61,7 +61,7 @@ def append_atf_node(file, atf_index, phy_addr):
     Append ATF DT node to input FIT dts file.
     """
     data = 'bl31_0x%08x.bin' % phy_addr
-    print >> file, '\t\tatf@%d {' % atf_index
+    print >> file, '\t\tatf_%d {' % atf_index
     print >> file, '\t\t\tdescription = \"ARM Trusted Firmware\";'
     print >> file, '\t\t\tdata = /incbin/("%s");' % data
     print >> file, '\t\t\ttype = "firmware";'
@@ -81,7 +81,7 @@ def append_fdt_node(file, dtbs):
     cnt = 1
     for dtb in dtbs:
         dtname = os.path.basename(dtb)
-        print >> file, '\t\tfdt@%d {' % cnt
+        print >> file, '\t\tfdt_%d {' % cnt
         print >> file, '\t\t\tdescription = "%s";' % dtname
         print >> file, '\t\t\tdata = /incbin/("%s");' % dtb
         print >> file, '\t\t\ttype = "flat_dt";'
@@ -91,17 +91,17 @@ def append_fdt_node(file, dtbs):
         cnt = cnt + 1
 
 def append_conf_section(file, cnt, dtname, atf_cnt):
-    print >> file, '\t\tconfig@%d {' % cnt
+    print >> file, '\t\tconfig_%d {' % cnt
     print >> file, '\t\t\tdescription = "%s";' % dtname
-    print >> file, '\t\t\tfirmware = "atf@1";'
-    print >> file, '\t\t\tloadables = "uboot@1",',
+    print >> file, '\t\t\tfirmware = "atf_1";'
+    print >> file, '\t\t\tloadables = "uboot",',
     for i in range(1, atf_cnt):
-        print >> file, '"atf@%d"' % (i+1),
+        print >> file, '"atf_%d"' % (i+1),
         if i != (atf_cnt - 1):
             print >> file, ',',
         else:
             print >> file, ';'
-    print >> file, '\t\t\tfdt = "fdt@1";'
+    print >> file, '\t\t\tfdt = "fdt_1";'
     print >> file, '\t\t};'
     print >> file, ''
 
@@ -111,7 +111,7 @@ def append_conf_node(file, dtbs, atf_cnt):
     """
     cnt = 1
     print >> file, '\tconfigurations {'
-    print >> file, '\t\tdefault = "config@1";'
+    print >> file, '\t\tdefault = "config_1";'
     for dtb in dtbs:
         dtname = os.path.basename(dtb)
         append_conf_section(file, cnt, dtname, atf_cnt)
